libs/gi/page-team/src/CharacterDisplay/Content.tsx

```diff
@@ -122,34 +140,73 @@ function TabNav({
       }}
     >
       {!hideTitle && (
-        <CardContent
-          sx={{
-            display: 'flex',
-            justifyContent: 'center',
-            pb: 0,
-            textShadow: '#000 0 0 10px !important',
-            position: 'relative',
-          }}
-        >
-          <BootstrapTooltip
-            title={
-              teamChar.description ? (
-                <Typography>{teamChar.description}</Typography>
-              ) : undefined
-            }
+          <CardContent
+            sx={{
+              display: 'flex',
+              justifyContent: 'center',
+              pb: 0,
+              textShadow: '#000 0 0 10px !important',
+              position: 'relative',
+            }}
           >
-            <Typography
-              variant="h6"
-              sx={{ display: 'flex', gap: 1, alignItems: 'center' }}
-            >
-              <PersonIcon />
-              <strong>{teamChar.name}</strong>
-              <Divider orientation="vertical" variant="middle" flexItem />
-              <CheckroomIcon />
-              {database.teams.getActiveBuildName(loadoutDatum)}
-            </Typography>
-          </BootstrapTooltip>
-        </CardContent>
+            <CardActionArea onClick={() => setEditMode(true)}>
+              <BootstrapTooltip
+                title={
+                  teamChar.description ? (
+                    <Typography>{teamChar.description}</Typography>
+                  ) : undefined
+                }
+              >
+                <Typography
+                  variant="h6"
+                  sx={{ display: 'flex', gap: 1, alignItems: 'center', justifyContent: 'center' }}
+                >
+                  <PersonIcon />
+                  <strong>{teamChar.name}</strong>
+                  <Divider orientation="vertical" variant="middle" flexItem />
+                  <CheckroomIcon />
+                  {database.teams.getActiveBuildName(loadoutDatum)}
+                </Typography>
+              </BootstrapTooltip>
+            </CardActionArea>
+            <ModalWrapper open={editMode} onClose={() => setEditMode(false)}>
+              <CardThemed>
+                <CardHeader
+                  title="Edit Loadout"
+                  action={
+                    <IconButton onClick={() => setEditMode(false)}>
+                      <CloseIcon />
+                    </IconButton>
+                  }
+                />
+                <Divider />
+                <CardContent>
+                  <Box
+                    display="flex"
+                    flexDirection="column"
+                    gap={2}
+                    sx={{ mt: 2 }}
+                  >
+                    <TextFieldLazy
+                      label="Loadout Name"
+                      placeholder='Loadout Name'
+                      value={teamChar.name}
+                      onChange={(loadoutName) => handleName(loadoutName)}
```

Comment:
more of a nit, can be simplified
```suggestion
                      onChange={handleName}
```